common/text-level-modifiers/text-level-modifiers-en.html

```diff
@@ -0,0 +1,84 @@
+---
+{
+	"title": "Text-Level Modifiers",
+	"language":"en",
+	"altLangPage":"text-level-modifiers-fr.html",
+	"breadcrumbs":
+		[
+			{
+				"title": "GCWeb home",
+				"link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html"
+			}
+		],
+	"secondlevel": false,
+	"dateModified": "2023-07-20",
+	"share": "true"
+}
+---
+
+<h3>CSS Class</h3>
+<dl class="dl-horizontal">
+	<dt><code>.stretched-link</code></dt>
+	<dd>Works in conjunction with position-relative for its stretch limit</dd>
+	<dt><code>.small</code></dt>
+	<dd>Decreases the font size of the element</dd>
+	<dt><code>mark</code></dt>
+	<dd>Highlights the text</dd>
+	<dt><code>.lead</code></dt>
+	<dd>Increases the font size and line height</dd>
+	<dt><code>[class*=text-]</code></dt>
+	<dd>Changes the color of the text</dd>
+	<dt><code>.no-wrap</code></dt>
+	<dd>Prevents the text from wrapping</dd>
+	<dt><code>.fnt-nrml</code></dt>
+	<dd>Resets the font size to the default</dd>
+</dl>
+<div class="row">
+	<div class="col-md-6">
```

Comment:
Remove this grid, that is not needed here we can use the full page available width

```suggestion
```